terraform/deploy_new_portscan_instance.sh

```diff
@@ -14,31 +14,53 @@ function usage {
     exit 1
 }
 
+function check_dependencies {
+    if [ -z "$(command -v terraform)" ] || \
+        [ -z "$(command -v aws)" ] || \
+        [ -z "$(command -v jq)" ]
+    then
+        echo "This script requires the following tools to run:"
+        echo "- terraform"
+        echo "- aws (AWS CLI)"
+        echo "- jq"
+        exit 1
+    fi
+}
+
 function redeploy_instances {
```

Comment:
I think a comment is needed here stating 
* What the arguments to this function are (`start` and `stop`)
* Whether the `start` to `stop` range is exclusive (`[start, stop)`) or inclusive (`[start, stop]`)

Reply:
I added some comments and addressed your concern in both a comment about the function, and as a note in the usage output in 85aeda2. Thanks for pointing out that it was ambiguous 🐧

Reply:
What you wrote is perfect.  Thanks for clearing that up.